src/main/java/org/ow2/proactive/procci/model/occi/infrastructure/ComputeBuilder.java

```diff
@@ -78,7 +91,7 @@ public ComputeBuilder() {
      *
      * @param cloudAutomation is the instance of the cloud automation model for a compute
      */
-    public ComputeBuilder(Model cloudAutomation) throws SyntaxException {
+    public ComputeBuilder cloudAutomationModel(Model cloudAutomation) throws SyntaxException {
```

Comment:
You right due to design error I had to transform the constructor into a method, I have correct the problem in the next pull request.